packages/angular/src/side-panel/side-panel.stories.ts

```diff
@@ -0,0 +1,139 @@
+        [showClose]="showClose"
+        [showDrawer]="showDrawer"
+        [active]="active"
+        [overlay]="overlay"
+        [variation]="variation"
+        [side]="side"
+        (close)="active = !active; close.emit()"
+      >
+        <div aiSidePanelTitle [condensed]="titleCondensed">Filter</div>
+        <div class="panel-content">Content</div>
+        <div aiSidePanelFooter>
+          <button ibmButton="secondary" size="lg">Cancel</button>
+          <button ibmButton size="lg">Initiate</button>
+        </div>
+      </ai-side-panel>
+      <div style="display: inline-block; position: relative; margin-left: 1rem; margin-right: 1rem">
+        <h2>Content</h2>
+        <button
+          ibmButton
+          *ngIf="variation === 'slide-in' || variation === 'slide-over'"
+          (click)="active = !active"
+        >
+          {{ active ? 'Deactivate' : 'Activate' }}
+        </button>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc a dapibus nulla. Fusce et
+          enim et elit rutrum interdum quis eu nulla. Nulla neque neque, condimentum eget
+          pellentesque sit amet, volutpat ac enim. Etiam id magna vel dolor condimentum imperdiet.
+          Vivamus eu pellentesque turpis, eget ultricies lectus. Vestibulum sodales massa non
+          lobortis interdum. Sed cursus sem in dolor tempus tempus. Pellentesque et nisi vel erat
+          egestas ultricies.
+        </p>
+        <p>
+          Etiam id risus nec mi laoreet suscipit. Phasellus porttitor accumsan placerat. Donec
+          auctor nunc id erat congue, tincidunt viverra diam feugiat. Donec sit amet quam vel augue
+          auctor posuere. Nunc maximus volutpat nulla vel vehicula. Praesent bibendum nulla at erat
+          facilisis sodales. Aenean aliquet dui vel iaculis tincidunt. Praesent suscipit ultrices mi
+          eget finibus. Mauris vehicula ultricies auctor. Nam vestibulum iaculis lectus, nec sodales
+          metus lobortis non.
+        </p>
+        <p>
+          Suspendisse nulla est, consectetur non convallis et, tristique eu risus. Sed ut tortor et
+          nulla tempor vulputate et vel ligula. Curabitur egestas lorem ut mi vestibulum porttitor.
+          Fusce eleifend vehicula semper. Donec luctus neque quam, et blandit eros accumsan at.
+        </p>
+      </div>
+      <ai-side-panel
+        *ngIf="side === 'right'"
+        [showClose]="showClose"
+        [showDrawer]="showDrawer"
+        [active]="active"
+        [overlay]="overlay"
+        [variation]="variation"
+        [side]="side"
+        (close)="active = !active; close.emit()"
+      >
+        <div aiSidePanelTitle [condensed]="titleCondensed">Filter</div>
+        <div class="panel-content">Content</div>
+        <div aiSidePanelFooter>
+          <button ibmButton="secondary" size="lg">Cancel</button>
+          <button ibmButton size="lg">Initiate</button>
+        </div>
+      </ai-side-panel>
+    </div>
```

Comment:
It seems like it should be possible to directly include this template in the story ... We should probably avoid adding custom demo components wherever possible (for demo/doc readability) ... If we really do need it we should add a warning comment to the story template